exercises/APE3/0301.tex

```diff
@@ -0,0 +1,157 @@
+		$y_1$ && 8,1 && 2,9 & \\
+		$z_1$ && 6,6 && 7,6 &
+	\end{tabular} 
+	
+	\paragraph{Step 2.} Strategy $y_{1}$ is weakly dominated by the randomized strategy $\frac{2}{3} x_{1} + \frac{1}{3} z_{1}$. Here we look at the rows, hence we are looking at player 1, hence we have to consider the payoffs of player 1. The weak domination is clear, since $8 \leq \frac{2}{3} \cdot 9 + \frac{1}{3} \cdot 6 = 9$ and $2 \leq \frac{2}{3} \cdot 1 + \frac{1}{3} \cdot 7 = 3$. The table becomes
+
+\begin{tabular}[h!]{l|ccccc}
+%		&&&&& \Large{$C_2$} &&& \\
+		&& $y_2$ && $z_2$ \\
+		\hline
+		$x_1$ && 9,4 && 1,3 & \\
+		$z_1$ && 6,6 && 7,6 &
+	\end{tabular} 
+	
+\paragraph{Step 3.} Strategy $z_{2}$ is weakly dominated by the pure strategy $y_{2}$. Here we look at the columns, hence we are looking at player 2, hence we have to consider the payoffs of player 2. The weak domination is clear, since $3 \leq 4$ and $6 \leq 6$. The table becomes
+
+\begin{tabular}[h!]{l|ccc}
+%		&&&&& \Large{$C_2$} &&& \\
+		&& $y_2$ \\
+		\hline
+		$x_1$ && 9,4 & \\
+		$z_1$ && 6,6 &
+	\end{tabular} 
+
+\paragraph{Step 4.} Strategy $z_{1}$ is strongly dominated by the pure strategy $x_{1}$.Here we look at the rows, hence we are looking at player 1, hence we have to consider the payoffs of player 1. The strong domination is clear, since $6 < 9$. The table becomes
+
+\begin{tabular}[h!]{l|ccc}
+%		&&&&& \Large{$C_2$} &&& \\
+		&& $y_2$ \\
+		\hline
+		$x_1$ && 9,4 &
+	\end{tabular} 
+	
+
+\vspace{10mm}
+
+We end this exercise with a note about randomized strategies. At step 2, we can guess the result with a little bit of mathematical intuition. We guess that $y_1$ is weakly dominated by a combination of $x_{1}$ and $x_{1}$ because $8 < 9$ and $2 < 7$. However, we don't have a robust method to find the coefficients. We can do so by defining the linear program
+
+
+        
+
+\begin{equation*}
+    \begin{aligned}
+    \underset{}{\text{min }} &
+    0 & & \\
+    \text{such that  } &
+        \lambda_{1} + \lambda_{2} = 1 & \\
+        & 8 \leq 9 \lambda_{1} + 6 \lambda_{2} &  \\
+        & 2 \leq \lambda_{1} + 7 \lambda_{2} & \\
+        & \lambda_{1}, \lambda_{2} \geq 0. &
+    \end{aligned}
+\end{equation*}
+   
+
+We can solve this graphically, see Figure \ref{fig:feasProb} on page \pageref{fig:feasProb}. Let us give color-labels to the constraints.
+\begin{enumerate}
+    \item The equality constraint $\lambda_{1} + \lambda_{2} = 1$ is the \textbf{blue} line, and since this is an equality constraint, we have to find a solution $\parent{\lambda_{1}, \lambda_{2}}$ on the blue line.
+    \item The inequality constraint $8 \leq 9 \lambda_{1} + 6 \lambda_{2}$ is the \textbf{green} line, and looking at the sign of the inequality, we have to be above the red line.
+    \item The inequality constraint $2 \leq \lambda_{1} + 7 \lambda_{2}$ is the \textbf{red} line, and looking at the sign of the inequality, we have to be above the green line.
+\end{enumerate}
+
+
+  
+
+
+\begin{figure}[h]
```

Comment:
floats are not allowed in the solution environment. Please use `\begin{center}...\captionof{figure}{...}...\end{center}` instead